internal/namespaces/autocomplete/autocomplete.go

```diff
@@ -51,8 +51,8 @@ type autocompleteScript struct {
 
 // autocompleteScripts regroups the autocomplete scripts for the different shells
 // The key is the path of the shell.
-func autocompleteScripts(ctx context.Context) map[string]autocompleteScript {
-	binaryName := core.ExtractBinaryName(ctx)
+func autocompleteScripts(ctx context.Context, basename string) map[string]autocompleteScript {
+	binaryName := basename
```

Comment:
why not use `binaryName` as function argument ?

Reply:
good question, because the arg is call basename but I can change it to binaryName